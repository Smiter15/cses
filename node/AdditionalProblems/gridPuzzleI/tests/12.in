50
37 33 50 48 40 38 39 35 45 30 49 41 41 34 39 35 34 38 43 40 45 37 36 46 41 37 34 38 43 42 42 46 42 40 40 40 48 38 33 50 38 50 38 36 32 37 40 35 50 44
31 29 34 40 30 40 45 38 39 42 36 40 50 45 29 47 42 49 43 42 32 45 44 43 35 36 50 43 32 39 30 50 30 47 39 45 33 37 45 32 38 45 37 50 35 47 44 39 44 50
