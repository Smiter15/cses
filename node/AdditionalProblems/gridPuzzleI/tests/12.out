...X.XXXXXXXXX.XXXXX.XXX.XXX.X.X.XXX.XX.XXXX.XXXXX
...X.XXXXX.XXX.XXXXX.XXX..XX.X.X.XXX..X.XX.X.XXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
X.XXXXXXXXXXXX.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
..XX.XXXXXXXXX.XXXXX.XXXXXXX.X.X.XXX.XX.XXXXXXXXXX
...X.XXXXX.XXX.XXXXX.XXXXXXX.X.X.XXX.XX.XXXXXXXXXX
...X.XXXXXXXXX.XXXXX.XXX.XXX.X.X.XXXXXX.XXXXXXXXXX
...X.XX.XX.XXX.XXXXX.XXXX.XX.X.X.XXX.XX.XXXX.XXXXX
X.XX.XXXXXXXXX.XXXXXXXXXXXXXXX.X.XXXXXXXXXXXXXXXXX
...X.XX..X.XXX.XXXXX.XXX..XX.X.X.XXX..X..X.X.XXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
..XX.XXXXXXXXX.XXXXX.XXX..XXXX.X.XXXXXXXXXXXXXXXXX
X.XX.XXXXXXXXX.XXXXXXXXXXXXX...X.X.X.XXXXXXXXXXXXX
......X..X..XXXXXXXXXXXXXXXXXXXXXXXXX.X..X.X.XX.XX
XXXXXXXXXXXXXX.XXXXX.XXX..XX...X.X.X..XXXXXXXXXXXX
......X..X..XXXXXXXXXXXXXXXXXXXXXXXXXXX..X.X.XX.XX
XXXXXXXXXXXXXX.XXXXX.XXX..XX...X.X.X..X..X.X.XX.XX
......X.....XX.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXX.XX..XXX..XX..XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.X.X.X..X..X.X.XX.XX
.....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXX.X.....XX.X.X...XX..XXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXX.X....X.X.X..X..X.X.XX.XX
....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX..X.....XX.X.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXX...X....X....X.X.XXXXXXXXXXXXXXX
.......XXXXXXXXXXXXXXXXXXXXXXXXXXXXX..X..X.X.X...X
XXXXXXX.....X..X.X....XXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXX....X...XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.X.X......XXXXXXXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX...X.X...X
X....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXX.......X.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXX.X.X........XXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXX....X.X.....XXXXXXXXXXX
..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX....X.X...X
XX..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX........X....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXX........X....X.....XXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX......X.....X
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
X...........X.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXX...X........X...XXXXXXXXXXXXXXXXXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXX.X...........X.....X
X...........X..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXX..........XXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXX.X....X..........XXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.X.....X
